packages/design/src/Colors.mdx

```diff
@@ -24,4 +24,135 @@ ensure you are using colors for their intended purpose.
   alongside color to ensure your content can be understood by everyone.
 </Banner>
 
+## Usage Guidelines
+
+### Typography
+
+Typographic elements in Jobber should use consistent colors to ensure
+readability.
+
+#### Heading
+
+Headings have a bold, dark color to cement their hierarchy.
+
+#### Text
+
+A slightly softer color is used for body text for greater readability.
+
+##### Text--Supporting
+
+Text that is relevant but less important can be lower-contrast to suggest its’
+reduced importance.
+
+### Interactive
+
+Use these colors in buttons and form controls to communicate the presence and
+meaning of interaction.
+
+#### Interactive
+
+The default color used for interactive elements.
+
+#### Destructive
+
+Use to signfify that an interaction will destroy something in the users’ account
+or workflow.
+
+#### Cancel
+
+Use to signify that an interaction will cancel or exit a workflow
+
+#### Disabled
+
+Use to signify that an interactive element is disabled.
+
+##### Disabled--Secondary
+
+Use when a disabled element needs more than one shade.
```

Comment:
This seems strange to me. Why would this be a use case?

Reply:
Elements like checkbox, radio, button, all have multiple tones:

![image](https://user-images.githubusercontent.com/39704901/116900843-73eabc00-abf6-11eb-9f7a-107105efda3d.png)
![image](https://user-images.githubusercontent.com/39704901/116900991-a1d00080-abf6-11eb-9119-e6a691c125a5.png)
![image](https://user-images.githubusercontent.com/39704901/116901073-ba401b00-abf6-11eb-9ff5-1af0aad342f8.png)

Reply:
Actually I just had an idea to make this more clear, though I still think we need the color. Will update shortly.